paths/reset/fc-scenarios.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Scenarios
+permalink: /reset/fc-01
+next-page: /reset/fc-02
+facilitator: false
+sidebar:
+  nav: "advanced"
+main-content: |  
+  This course is comprised of a handful of scenarios that you might find yourself in while working with Git. Before getting started with the scenarios though, you should perform the following:
+
+    - Create a test repository on [GitHub.com](https://www.github.com)
+    - Clone the test repository to your local machine
+
+  ## New UI Addition
+  If you have taken some of the other courses provided on this site, you might be familiar with the `Show Me How` and `Tell Me Why` drop-downs. In this course we added two new drop-downs, `I didn't push` and `I pushed`. When using the various Git tools to get out of a pickle, the best tool for the job is typically dependent on if you pushed your commits to your remote (or not). Each scenario (for the most part), provides the best command to use based on if your pushed your commits or not.
+
+  ![](/on-demand/images/push-dropdowns.png){: .align-center}
+
+  The `I pushed` instructions assume that you have already `push`ed your commits to your remote branch. If you haven't `push`ed the changes, you probably won't see the expected changes on your remote.
+
+  ## Setting Up Your Scenario Environment
+
+  Unlike our other courses, this class isn't linear which means before you can begin working on any scenario, you are going to need to do a little bit of setup to get a 'testing environment' ready.
+
+   1. Create a repository on GitHub.com and `clone` it to your desktop.
+
+       With a remote repository created, we need to populate our repository with some files and commits.
+
+   1. Before creating new files, you should probably create a new branch, maybe call it something like ```test```.
+
+  With a branch created, its time to create those files, thankfully we have some handy scripts that will automatically generate files and commits (with commit messages!) that will make your life a little easier.
+
+   - **Bash:** ```for d in {1..6}; do touch file$d.md; git add file$d.md; git commit -m "adding file $d"; done```
+   - **PowerShell:** ```for ($d=1; $d -le 6;$d++) { touch file$d.md; git add file$d.md; git commit -m "adding file$d.md";}```
+
+  If you created a repository on GitHub.com, cloned it, created a branch, and created files and commits, you are ready to rock and roll! Sorry about the GIANT block of text to get setup :cry:.
+
+  ## Scenarios
+  - [Committed Something on Accident](/on-demand/reset/fc-05)
```

Comment:
1. Made this consistent
1. this highlighted an issue with the descriptions not lining up with the outline 
1. great catch 🎣